tests/test_consumer/test_extractors/test_extractors_operation_spark.py

```diff
@@ -302,3 +304,62 @@ def test_extractors_extract_operation_spark_job_finished(
         started_at=None,
         ended_at=ended_at,
     )
+
+
+def test_extractors_extract_operation_spark_job_sql_query():
+    now = datetime(2024, 7, 5, 9, 6, 29, 462000, tzinfo=timezone.utc)
+    run_id = UUID("01908224-8410-79a2-8de6-a769ad6944c9")
+    operation_id = UUID("01908225-1fd7-746b-910c-70d24f2898b1")
+
+    operation = OpenLineageRunEvent(
+        eventType=OpenLineageRunEventType.START,
+        eventTime=now,
+        job=OpenLineageJob(
+            namespace="anything",
+            name="mysession.execute_some_command",
+            facets=OpenLineageJobFacets(
+                jobType=OpenLineageJobTypeJobFacet(
+                    processingType=OpenLineageJobProcessingType.BATCH,
+                    integration="SPARK",
+                    jobType="SQL_JOB",
+                ),
+                sql=OpenLineageSqlJobFacet(query="select id, name from schema.table where id = 1"),
```

Comment:
Let's use `"""` here to test indentation removal

Reply:
OK.
I am thinking